app/controllers/todos_controller.rb

```diff
@@ -1,25 +1,51 @@
 class TodosController < ApplicationController
   def index
-    render :json => get_paginated_todos
+    todos = PaginateItemsService.run!(
+      :per => params[:per],
+      :page => params[:page],
+      :items => todo_list_joined_with_users
+    )
+
+    render :json => { 
+      :todos => todos,
+      :total_record_count => TodoList.count
+    }
+    
```

Comment:
orinak `GetTodosService`
very veradardznuma srazu.
```
{
  todos: todos_data,
  total_record_count: todos_data.count
}
```

is `indexumy` prosto stugums

```
service = GetTodosService.run(...)

if service.valid?
  render json: service.result
else
  render json: service.errors, status: 400
end
```

1) Hinch petqa servicum srazu veradardzru, controllerum starachil  DB zapros anumel.
2) Ete cod-y shata erkar, apa texapoxe Service, ete che tox mna controllerum.